WooCommerce/UITestsFoundation/Screens/Payments/PaymentsScreen.swift

```diff
@@ -25,4 +25,10 @@ public final class PaymentsScreen: ScreenObject {
         cardReaderManualsButton.tap()
         return try CardReaderManualsScreen()
     }
+
+    @discardableResult
+    public func verifyPaymentsScreenLoaded() throws -> PaymentsScreen {
+        XCTAssertTrue(isLoaded)
+        return self
+    }
```

Comment:
WDYT of extending the `expectedElementGetters` for this screen?

```swift
expectedElementGetters: [ collectPaymentButtonGetter, cardReaderManualsButtonGetter ],
```

The `init` still looks for the first element only, and for the assertion two elements will be checked.

Reply:
yup we can! added in: https://github.com/woocommerce/woocommerce-ios/pull/9384/commits/f941d9ecb1d62732863c60c32189f6639f99ee6a